pandas/core/generic.py

```diff
@@ -8027,7 +8027,8 @@ def resample(
         level : str or int, optional
             For a MultiIndex, level (name or number) to use for
             resampling. `level` must be datetime-like.
-        origin : {'epoch', 'start', 'start_day'}, Timestamp or str, default 'start_day'
+        origin : {'epoch', 'start', 'start_day', 'end', 'end_day'}, Timestamp \
+        or str, default 'start_day'
```

Comment:
i think you have to indent this 2nd line

Reply:
Here has a line-continuation at the end of 1st line, will this break the display if I indent the 'or str, default "start_day"' with 4 spaces?

Reply:
yeah saw that but we don't usually use continuation like that

it will render if you indent on 2nd line

Reply:
ok